tests/System.Buffers.Experimental.Tests/MemoryTests.cs

```diff
@@ -90,5 +90,161 @@ static void TestLifetime(OwnedBuffer<byte> owned)
                 var span = copyStoredForLater.Span;
             });
         }
+
+        [Fact]
+        public unsafe void IsSliceOf_PointerVersusArraySpansNeverMatch()
+        {
+            ulong[] data = new ulong[8];
+            fixed (ulong* ptr = data)
+            {
+                Span<ulong> arrayBased = data;
+                Span<ulong> pointerBased = new Span<ulong>(ptr, 8);
+
+                // comparing array to pointer says "no"
+                Assert.False(arrayBased.IsSliceOf(pointerBased));
```

Comment:
This has been fixed here: https://github.com/dotnet/corefxlab/pull/830/commits/fb3fdd8db53d65ee18dc97370efe6803f1dc0d8c